treeherder/perf/management/commands/perf_datum_pk_migration.py

```diff
@@ -0,0 +1,78 @@
+"""IMPORTANT!
+This subcommand isn't intended for regular use.
+Its purpose is to fill up performance_datum new table with bigint primary keys.
+It should be runned after migration perf.0043_perf_datum_bigint_table
+and before migration perf.0044_perf_datum_table_swap
```

Comment:
how is this intended to be run?  manually on a database?  or is this kicked off automatically?

Reply:
It should be run manually. This script only helps with copying all data to the new table, but perf.0044 migration will copy the remaining data anyway.
I think Django won't use a transaction for the migration, but it should be convenient not to loss data in the final migration (as you mentioned in another comment).